src/paragraph/paragraph.tsx

```diff
@@ -51,7 +53,7 @@ class Paragraph extends PureComponent<ParagraphProps> {
 
     return (
       <div className={cc(className)} role="presentation">
-        <Text tag={TextTagType.PARAGRAPH} newlineToBr>
+        <Text {...(itemProp && { itemProp })} tag={TextTagType.PARAGRAPH} newlineToBr>
```

Comment:
Can you do spread this since this itemProp can be `null`?

Reply:
in typescript > 3.2.4, yes